cmd/orb.go

```diff
@@ -288,6 +331,55 @@ func releaseOrb(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func devOrb(cmd *cobra.Command, args []string) error {
+	// Ensure the `dev:` tag is prefixed to the label, no matter what
+	label := fmt.Sprintf("dev:%s", strings.TrimPrefix(args[3], "dev:"))
+
+	ctx := context.Background()
+	response, err := api.OrbPublish(ctx, Logger, args[0], args[1], args[2], label)
+
+	if err != nil {
+		return err
+	}
+
+	if len(response.Errors) > 0 {
+		return response.ToError()
+	}
+
+	Logger.Info("Orb published")
+	return nil
+}
+
+var validSegments = []string{"major", "minor", "patch"}
+
+func validateSegmentArg(args []string) error {
+	label := args[3]
+	for _, segment := range validSegments {
+		if label == segment {
+			return nil
+		}
+	}
+	return fmt.Errorf(`expected %s to be one of "major", "minor", or "patch"`, label)
+}
+
+func incrementOrb(cmd *cobra.Command, args []string) error {
+	if err := validateSegmentArg(args); err != nil {
+		return err
+	}
+
+	v2, err := api.IncrementOrb(context.Background(), Logger, args[0], args[1], args[2], args[3])
+	if err != nil {
+		return err
+	}
+
+	Logger.Infof("Orb %s bumped to %s\n", args[1]+"/"+args[2], v2)
```

Comment:
can you do the string manipulation in the format string? `("Orb %s/%s bumped to %s\n", args[1], args[2], v2)`

No need to be creating extra strings here with `+`.